mapswipe_workers/tests/test_archive_project.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from mapswipe_workers import auth
+from mapswipe_workers.firebase_to_postgres import archive_project
+
+
+class TestArchiveProject(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        """Set up before tests are run."""
+        cls.project_id = "test_archive_project"
+        # TODO: Create project
+        archive_project.archive_project(cls.project_id)
+
+    @classmethod
+    def tearDownClass(cls):
+        """Tear down after tests are run."""
+        pg_db = auth.postgresDB()
+        sql_query = "".format(cls.project_id)
```

Comment:
I'm not sure what happens at this step or what is intended.

Reply:
Proper tests will be implemented in the unittest branch.